streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/core/task/FlinkRESTAPIWatcher.java

```diff
@@ -505,6 +505,8 @@ and the status is not obtained this time (flink rest server is closed),
         log.error(
             "FlinkRESTAPIWatcher query previous state was canceling and stopFrom NotFound,savePoint expired!");
         savePointService.expire(application.getId());
+        FlinkAppState flinkAppState = FlinkAppState.of(application.getState());
```

Comment:
Hello, thanks for your contribution. We have checked and found that the job status of the assignment is CANCELED, and it was canceled through StreamaPark. We consider CANCELED to be a normal stoppage status (manually triggered by the user). Our current policy is not to issue any alerts for statuses such as CANCELED and SUCCESS. Only when the assignment FAILED，LOST，CANCELED (not stopped through Streamapark) will an alert be triggered.

Reply:
Sorry for late reply. I've been quite busy these days. Will resubmit the code in the next few days...

Reply:
> Sorry for late reply. I've been quite busy these days. Will resubmit the code in the next few days...

I look forward to your driving

Reply:
Done. PTAL @wolfboys